src/components/Zap/ZapAction/ZapActionFaqAccordionHeader.tsx

```diff
@@ -0,0 +1,10 @@
+import { Typography } from '@mui/material';
+import { AccordionHeader } from 'src/components/AccordionFAQ';
+
+export const ZapActionFaqAccordionHeader = () => {
+  return (
+    <AccordionHeader sx={{ margin: 0, marginBottom: '4px', marginLeft: '8px' }}>
+      <Typography variant="title2XSmall">How to participate</Typography>
```

Comment:
Would need to be a translation

Reply:
Content on that page is being pulled from strapi which is also not multi-language, yet.

Reply:
agree